packages/mdc-select/_mixins.scss

```diff
@@ -147,7 +147,10 @@
   }
 }
 
-@mixin mdc-select-dd-arrow-svg-bg_($fill-hex-number: 000000, $opacity: .54) {
+@mixin mdc-select-dd-arrow-svg-bg_($fill-hex-number: #000000, $opacity: .54) {
```

Comment:
* Remove the default value for the first parameter since we always specify it
* Should the default opacity be represented in a variable? Should we also always specify that in the mixin invocation rather than providing a default? (Currently we pass 1 parameter in one invocation, and 2 parameters in the other)